charmhelpers/contrib/storage/linux/ceph.py

```diff
@@ -399,16 +400,28 @@ def __init__(self, service, name, pg_num=None, replicas=2,
 
     def create(self):
         if not pool_exists(self.service, self.name):
+            nautilus_or_later = cmp_pkgrevno('ceph-common', '14.2.0') >= 0
             # Create it
-            cmd = ['ceph', '--id', self.service, 'osd', 'pool', 'create',
-                   self.name, str(self.pg_num)]
+            if nautilus_or_later:
```

Comment:
I wonder if it would be nicer or more clear to do something like:

```python3
cmd = ['ceph', '--id', self.service, 'osd', 'pool', 'create']
if nautilus_or_later:
    cmd.append(
        '--pg-num-min={}'.format(
             min(AUTOSCALER_DEFAULT_PGS, self.pg_num))
cmd.extend([self.name, str(self.pg_num)])
```

Reply:
I'm kinda split on that one - I like the clarity of all of the command that will be executed being between [] even if that does mean duplication of a couple of lines of code.

Reply:
I was thyinking about the potential for future conditions where we want more checks like these; however, we haven't had many changes in this code in years so probably better to be clear now, and worry about the future if things change.